### PR TITLE
NLP metrics

### PR DESCRIPTION
**Pull request by lalmei**
### General Checklist

* [x] Tests added for this feature/bug
      if it was a bug, test must cover it.
* [x] Conform by the style guides, by using formatter
* [x] Documentation updated
* [x] (optional) Please add a label to your PR

### CODE REVIEW

**Comment by lalmei** [comment]
Still need to add a notebook for the overall metrics, and some more tests

File: src/whylogs/core/metrics/nlp_metrics.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from typing import List, Union
+
+import jiwer
```
Comment (bernease) [inline src/whylogs/core/metrics/nlp_metrics.py]: We decided to try / import for this in our meeting.

**Review by bernease — APPROVED** [review]
LGTM

File: src/whylogs/core/metrics/nlp_metrics.py (hunk 1 above)
Comment (lalmei) [reply src/whylogs/core/metrics/nlp_metrics.py]: it was done

**Pull request merged**